source/material_model/rheology/elasticity.cc

```diff
@@ -371,7 +371,7 @@ namespace aspect
                                          const SymmetricTensor<2,dim> &stress,
                                          const double shear_modulus) const
       {
-        const SymmetricTensor<2,dim> edot = 2. * (deviator(strain_rate)) + stress /
+        const SymmetricTensor<2,dim> edot = deviator(strain_rate) + 0.5*stress /
```

Comment:
@bobmyhill - I think we should add a comment here to indicate why multiplying by 0.5 is necessary. Feel free to significantly modify the suggestion below. After, that this should be good to go.
```suggestion
        // Multiplying the elastic stress below by a factor of 0.5 is necessary to be consistent with 
        // with ASPECT's definition of stress, where stress = 2 * viscosity * strain_rate_second_invariant.
        const SymmetricTensor<2,dim> edot = deviator(strain_rate) + 0.5*stress /
```